Dockerfile

```diff
@@ -83,7 +45,7 @@ RUN apk add --no-cache --virtual .crate-rundeps openjdk8-jre-base python3 openss
     && mkdir /crate \
     && tar -xf crate-$CRATE_VERSION.tar.gz -C /crate --strip-components=1 \
     && ln -s /usr/bin/python3 /usr/bin/python \
-    && cp -f /usr/local/lib/*.so /crate/plugins/sigar/lib/ \
+    && cp -f /usr/lib/libsigar-amd64-linux.so /crate/plugins/sigar/lib/ \
```

Comment:
do you still need to copy it into the crate plugins folder? doesn't Crate look for the library also in the system libs?